frontend/src/app/AdminNavBar/AdminNavBar.tsx

```diff
@@ -143,6 +219,12 @@ export const AdminNavBar = ({ isMenuOpen }: AdminNavBarProps): JSX.Element => {
           {NAV_LINKS.map((link, index) => (
             <AdminNavBarLink key={index} {...link} />
           ))}
+          <WhatsNewNavBarTab
+            label={WHATS_NEW_LABEL}
```

Comment:
nit: should we just hardcode the label into the component instead of passing it in as a prop